### PR TITLE
add multi-ingress template

### PR DESCRIPTION
**Pull request by ElkhaberiBF**
## What this PR does / why we need it:

Adding helm template to create multi-able ingresses for one release, it will help us to create a separate ingress for VPN white-listed endpoints 

## Checklist

- [ ] Meaningful PR title
- [ ] Updated README
- [ ] Github actions are passing
- [ ] I have added the Jira task

### CODE REVIEW

**Review by 13sh — DISMISSED** [review]

**Review by ayyash-b7t — APPROVED** [review]

File: charts/service/values.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ image:
   # -- Overrides the image tag whose default is the chart appVersion.
 
 imagePullSecrets: []
-nameOverride: "{{ .Release.Name }}"
```
Comment (13sh) [inline charts/service/values.yaml:14]: why was this changed? all older ingresses would lose their name and break

Comment (ElkhaberiBF) [reply charts/service/values.yaml:14]: @13sh It's not applicable to set expression in values.yaml.
"Caught by Github checks"

Comment (13sh) [reply charts/service/values.yaml:14]: Does this mean the values currently used will change or not?

Comment (ElkhaberiBF) [reply charts/service/values.yaml:14]: Nothing will change, we override it by  helmvalues in gitops

**Review by 13sh — APPROVED** [review]
